src/extension.ts

```diff
@@ -274,12 +309,20 @@ function registerCompletionProvider(languageSelector: string, classPrefix = ''):
 }
 
 /**
- * Register Hover Provider
+ * Registers a hover provider for the specified language.
  *
- * @param languageSelector
+ * @param {string} languageSelector - The language identifier to register the hover provider for.
+ * @returns {Disposable} A disposable object that can be used to unregister the hover provider.
  */
 function registerHoverProvider(languageSelector: string): Disposable {
     return languages.registerHoverProvider(languageSelector, {
+        /**
+         * Provides hover information for the given position in the document.
+         *
+         * @param {TextDocument} document - The document in which the hover was invoked.
+         * @param {Position} position - The position at which the hover was invoked.
+         * @returns {Hover | null} A Hover object containing the hover information, or null if no information is available.
+         */
         provideHover(document: TextDocument, position: Position): Hover | null {
             _output.appendLine(`In hover, position is ${position.line}:${position.character}`);
```

Comment:
Do you wanna clean up some of these outputs I added for debugging hover? I don't think we need it outputting all this anymore now that it works.